src/main/java/tdl/datapoint/infra_events/EventsAlertHandler.java

```diff
@@ -73,9 +73,9 @@ private static AmazonSQS createSQSClient(String serviceEndpoint, String signingR
     }
 
     @Override
-    public String handleRequest(Map<String, Object> s3EventMap, Context context) {
+    public String handleRequest(Map<String, Object> eventMap, Context context) {
         try {
-            handleS3Event(S3BucketEvent.from(s3EventMap, jsonObjectMapper));
+            handleS3Event(S3BucketEvent.from(eventMap, jsonObjectMapper));
```

Comment:
Some event detection needs to happen at this point, some tests need to be added for this

Reply:
This part is still up in the air so will give it a think

Reply:
Mostly done via d1ca431